src/error-handling/thiserror-and-anyhow.md

```diff
@@ -46,23 +44,16 @@ fn main() {
 
 <details>
 
-## `thiserror`
-
-- The `Error` derive macro is provided by `thiserror`, and has lots of useful
-  attributes to help define error types in a compact way.
-- The `std::error::Error` trait is derived automatically.
-- The message from `#[error]` is used to derive the `Display` trait.
-
-## `anyhow`
-
 - `anyhow::Error` is essentially a wrapper around `Box<dyn Error>`. As such it's
   again generally not a good choice for the public API of a library, but is
   widely used in applications.
 - `anyhow::Result<V>` is a type alias for `Result<V, anyhow::Error>`.
-- Actual error type inside of it can be extracted for examination if necessary.
-- Functionality provided by `anyhow::Result<T>` may be familiar to Go
-  developers, as it provides similar usage patterns and ergonomics to
-  `(T, error)` from Go.
+- Actual error type inside of it can be extracted for examination if necessary,
+  much like `std::any::Any`.
```

Comment:
We never explain `Any` to students do we? Probably don't need to mention in the speaker notes, since it's an unnecessary tangent.

Reply:
I think it's worthwhile to mention because many students come from backgrounds in languages where upcasting/downcasting is more common. The fact that we can downcast if we happen to know the concrete underlying type here means we're not simply out of luck compared to using an error enum if we want to do something with one specific possible error, which could inform choices of which style to use.

Reply:
We have the option of `# More to Explore` in the speaker notes for things we don't intend to talk about but might be of interest. Maybe that helps here?

Reply:
Yeah moving it to More to Explore seems reasonable to me.